cmd/workspace/apps/overrides.go

```diff
@@ -0,0 +1,61 @@
+package apps
```

Comment:
tldr: support:

 `databricks app create --manifest ./manifest.yaml --resources ./resources.yaml`
 `databricks app create --json '{"manifest".:{......'`

throws an error if neither manifest nor json arguments are supplied

Reply:
update: also supports inline yaml string (like the --json flag does)